hub/management/commands/import_area_fuel_poverty_data.py

```diff
@@ -33,6 +33,7 @@ class Command(
                 "source_type": "xlxs",
                 "data_url": data_url,
                 "table": "areadata",
+                "is_shadable": False,
```

Comment:
I think this is ok to filter by?

Reply:
although we're not using this dataset anymore in favour of the EFPC one.

Reply:
You're right! I've removed it, either way